### PR TITLE
feature: add employee access

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: fabric/dataclient.go [hunk 1]
```diff
@@ -585,6 +596,178 @@ func createRejectRoute(rid, path, host string, privs []interface{}) *eskip.Route
...
+			},
+		},
+		BackendType: eskipBackend.Type,
+		Backend:     eskipBackend.backend,
+		LBAlgorithm: eskipBackend.lbAlgorithm,
+		LBEndpoints: eskipBackend.lbEndpoints,
+	}
+
+	// add optional ratelimit only default ratelimit
+	if m.Ratelimit != nil {
```
Comment (marcinzaremba) [inline fabric/dataclient.go:675]: Following three sections seems to be a direct duplication of sections from `createServiceRoute`. Maybe we can somehow share these parts? WDYT?

Comment (szuecs) [reply fabric/dataclient.go:675]: there is quite some overlap in the route creations. I would like to refactor when we are done with the features.
I hope we can de-duplicate more than that :)

**Comment by marcinzaremba** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**